### PR TITLE
Correct grammar and unclear phrasing in docs

### PR DESCRIPTION
**Pull request by andrewdushane**
These changes don't change the content of the docs, just fix grammar and clarify where needed.

One exception is the addition of a paragraph about Create React App in the main readme, seemed like a really common/obvious question wrt Gluestick, but would be happy to take out if not appropriate for this PR.

### CODE REVIEW

**Comment by andrewdushane** [comment]
[clabot:check]

File: docs/Commands.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 ```bash
 gluestick new <YOUR_APPLICATION_NAME>
 ```
 
 Available options:
 
-* `-d, --dev <path>` - Relative path from inside crated directory to development version of gluestick
-* `-n, --npm` - Use npm instead of yarn for install dependencies
-* `-s, --skip-main` - Gluestick will not generate main app
+* `-d, --dev <path>` - Relative path from inside the created directory to the development version of gluestick
```
Comment (christinebrass) [inline docs/Commands.md:17]: Can we also make the following change here? 

~Relative path~Use a relative path...

File: docs/Commands.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
+* `-n, --npm` - Use npm instead of yarn to install dependencies
+* `-s, --skip-main` - Gluestick will not generate the main app
```
Comment (christinebrass) [inline docs/Commands.md:19]: Can we also change this to:

`- Skip generation of the main app`

File: docs/Commands.md [hunk 3]
```diff
@@ -159,23 +162,23 @@ gluestick test --watch Home
...
+Watches and applies changes from the gluestick package to the current project
 
 ```bash
 gluestick watch
 ```
 
 ### `gluestick reset-hard`
 
-Removes gluestick dependency project clean build, cache and reinstalls dependencies
+Removes core GlueStick dependencies and cahce, and reinstalls GlueStick dependencies
```
Comment (christinebrass) [inline docs/Commands.md:181]: ~cahce~ -> cache

File: docs/Configuration.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Configuration
 
-There are few files which allows to configure GlueStick:
+There are few files which allow configuration of Gluestick:
```
Comment (christinebrass) [inline docs/Configuration.md:3]: Glue~s~Stick

File: docs/Configuration.md [hunk 5]
```diff
@@ -13,32 +13,34 @@ There are few files which allows to configure GlueStick:
...
 
-GlueStick itself will look only for those properties:
+GlueStick itself will look only for these properties:
 * `proxies` - setup additional proxies
 * `httpClient` - configure http client
 * `headContent` - configure page `<title>` element
 * `logger` - configure logger
 
-However, you can put your own properties here and read them by importing this files via alias `config/application`.
+However, you can put your own properties here and read them by importing this files via the alias
```
Comment (christinebrass) [inline docs/Configuration.md:40]: "this file~s~"

File: docs/Configuration.md [hunk 6]
```diff
@@ -82,14 +84,14 @@ export default config;
...
-To modify GlueStick config use `src/gluestick.config.js` file. It must return function or array of functions
+To modify the GlueStick config use `src/gluestick.config.js`. It must return a function or array of functions
 as a `default` export:
 ```
 export default config => config;
 // or
 export default [config => config];
 ```
-This function accepts gluestick config (`object`) as a first argument and must return modified
+This function accepts the gluestick config (`object`) as a first argument and must return the modified
```
Comment (christinebrass) [inline docs/Configuration.md:94]: I wonder if we should just substitute all uses of `gluestick` with `GlueStick` in this file (unless it's the command)

File: docs/Configuration.md [hunk 7]
```diff
@@ -102,15 +104,15 @@ export default config => ({
   }
 })
 ```
-To modofy path to asset files (previously known as `assetPath`), define `publicPath` and assign a value in Gluestick config, for example:
+To modify the path to asset files (previously known as `assetPath`), define `publicPath` and assign a value in the Gluestick config, for example:
```
Comment (christinebrass) [inline docs/Configuration.md:107]: Glue~s~Stick

File: docs/Configuration.md [hunk 8]
```diff
@@ -129,22 +131,22 @@ or
...
 
 If you want to use `DllPlugin`, you firstly need to execute `gluestick build --vendor` (for production add `NODE_ENV=production` when running the command), before building any other app.
-This command will create files (including your vendor DLL bundle) under `build/assets/dlls`. Now if you `build` or `start` an app or whole project, it will use vendor DLL bundle.
+This command will create files (including your vendor DLL bundle) under `build/assets/dlls`. Now if you `build` or `start` an app or the whole project, it will use the vendor DLL bundle.
 
-`DllPlugin` is the essential part that allows for app builds parallelisation.
-In order to split app builds you, firstly need to have vendor DLL bundle compiled and available for each
-build thread/process, then to build a single app use `gluestick build --client -A /<appName>` command.
+`DllPlugin` is the essential part that allows for app build parallelisation.
+In order to split app builds you, firstly you need to have the vendor DLL bundle compiled and available for each
```
Comment (christinebrass) [inline docs/Configuration.md:148]: Change to:

"In order to split app builds, you'll first need to have the vendor DLL[...]"

File: packages/gluestick-cli/README.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
 # `gluestick-cli`
-Gluestick CLI is an command line interface for `gluestick` package. It exposes few commands from your global packages, and proxies rest of them to your [local distribution of gluestick](../packages/gluestick).
+Gluestick CLI is an command line interface for the `gluestick` package. It exposes the few commands from your global packages, and proxies the rest of them to your [local distribution of gluestick](../packages/gluestick).
```
Comment (christinebrass) [inline packages/gluestick-cli/README.md:2]: Glue~s~Stick

File: packages/gluestick-cli/README.md [hunk 10]
```diff
@@ -20,23 +20,23 @@ Available options:
 
 ### `gluestick reinstall-dev`
 
-Reinstall gluestick dependency project
+Reinstall Gluestick dependencies
```
Comment (christinebrass) [inline packages/gluestick-cli/README.md:23]: Glue~s~Stick

File: packages/gluestick-cli/README.md [hunk 11]
```diff
@@ -20,23 +20,23 @@ Available options:
...
 
 ```bash
 gluestick reinstall-dev
 ```
 
 ### `gluestick watch`
 
-Watches and applies changes from gluestick package to current project
+Watches and applies changes from the Gluestick package to the current project
```
Comment (christinebrass) [inline packages/gluestick-cli/README.md:31]: Glue~s~Stick

File: packages/gluestick-cli/README.md [hunk 12]
```diff
@@ -20,23 +20,23 @@ Available options:
...
 
 ```bash
 gluestick watch
 ```
 
 ### `gluestick reset-hard`
 
-Removes gluestick dependency project clean build, cache and reinstalls dependencies
+Removes Gluestick dependencies and does a clean build
```
Comment (christinebrass) [inline packages/gluestick-cli/README.md:39]: Glue~s~Stick

File: packages/gluestick-plugin-no-fouc/README.md [hunk 13]
```diff
@@ -3,7 +3,7 @@
 Removes Flash of Unstyled Content (FOUC) in development.
 
 ## How it works
-All styles will be extracted to css file using `ExtractTextWebpackPlugin` then, the file will be linked by server/renderer and added to `<head>` element. It sits side-by-side with `style-loader`, so you can still use HMR.
+All styles will be extracted to a css file using `ExtractTextWebpackPlugin` then, the file will be linked by server/renderer and added to `<head>` element. It sits side-by-side with `style-loader`, so you can still use HMR.
```
Comment (christinebrass) [inline packages/gluestick-plugin-no-fouc/README.md:6]: "CSS" for consistency with the above changes?

**Review by christinebrass — COMMENTED** [review]
Hey @andrewdushane! I found a couple areas of improvement/typo fixes. Let me know if you have any questions, and thanks so much for this!!

**Comment by christinebrass** [comment]
[clabot:check]

**Review by christinebrass — APPROVED** [review]

**Pull request merged**